pkg/service/controller/servingcert/secret_updating_controller.go

```diff
@@ -161,29 +163,30 @@ func (sc *ServiceServingCertUpdateController) processNextWorkItem() bool {
 // syncSecret will sync the service with the given key.
 // This function is not meant to be invoked concurrently with the same key.
 func (sc *ServiceServingCertUpdateController) syncSecret(key string) error {
-	obj, exists, err := sc.secretCache.GetByKey(key)
+	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	sharedSecret, err := sc.secretLister.Secrets(namespace).Get(name)
+	if kapierrors.IsNotFound(err) {
+		glog.V(4).Infof("Secret has been deleted %v", key)
```

Comment:
This message isn't accurate.   You sure you want to keep it?

Reply:
this was exists == false before